Dockerfile

```diff
@@ -1,8 +1,14 @@
 FROM gcc:7.2
 
-ENV FFMD_REPO=https://github.com/FreifunkMD/site-ffmd.git
-ENV FFMD_VERSION=tags/v0.38-beta.2
-ENV GLUON_VERSION=origin/v2016.2.x
+ARG FFMD_REPO=https://github.com/FreifunkMD/site-ffmd.git
+ARG FFMD_VERSION=tags/v0.38-beta.2
+ARG GLUON_VERSION=origin/v2016.2.x
```

Comment:
Hängt die Gluon-Version nicht eigentlcih von der FFMD-Version ab? Ich meine, dass das Build-Script da entsprechend nachschaut und dann den passenden Branch auscheckt. Sollte das im Dockerfile auch passieren?

Reply:
Ja, welcher Gluon-Release da genommen werden sollen, steht in der site.mk oder in der site.conf drin. Dazu bin ich aber nicht fähig, weil ich Bash nicht in den Maße beherrsche.

Reply:
@johannwagner  wenn Du magst, können wir da gern mal gemeinsam drauf starren, ich spreche bash...